fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -450,6 +451,10 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = QUERY_TIMEOUT)
     public int queryTimeoutS = 300;
 
+    // query timeout in second.
+    @VariableMgr.VarAttr(name = ANALYZE_TIMEOUT)
+    public int analyzeTimeoutS = 43200;
```

Comment:
does need forward to master?

Reply:
Is it too long for the analyze stmt?

Reply:
> does need forward to master?

Yes, I think it does need to forward, will fix it.

Reply:
> Is it too long for the analyze stmt?

I think 12 hours is OK here. When user send analyze table/db command with sync, I think it shouldn't timeout.  The user could manually kill it. Or, they can analyze async with background tasks.